<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>723</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1088</width>
    <height>723</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1088</width>
    <height>723</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Inserisci abbonamento</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1088</width>
     <height>723</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>InserimentoNuovoAbbonamento.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="id">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>147</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 15px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataInizio">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>200</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 15px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tipologia">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>260</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 15px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataScadenza">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>320</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 15px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_conferma">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>400</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>CONFERMA</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="InserimentoAbbonamento.qrc"/>
 </resources>
 <connections/>
</ui>
